### PR TITLE
Feat buttons

### PR DESCRIPTION
**Pull request by hinanshisuthar**
Button Component Added
- Primary
- Link
- Icon Button
- Floating Action

https://user-images.githubusercontent.com/88201664/152681308-a9678d72-05a1-44bf-9c24-8d7a1d33eb3a.mp4

### CODE REVIEW

File: components/button.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+/* link buttons */
+
+/* .link {
+    border-bottom: 2px solid var(--cta-color);
+}
+
+.link:hover {
+    border-bottom: 2px solid var(--hover-color-links);
+} */
```
Comment (aryan-13) [inline components/button.css]: Commented code can be removed

Comment (omgaikwad) [inline components/button.css]: you can remove these comments

**Review by aryan-13 — COMMENTED** [review]
Else the code looks good

**Review by omgaikwad — COMMENTED** [review]
The code looks good, the link button hover looks cool!

**Pull request merged**